src/main/ccl/cclutmock_table.inc

```diff
@@ -427,12 +437,41 @@ subroutine cclutRemoveMockTable(cclutTableName)
     set cclutMockTablePos = cclut::cclutGetMockTableIndex(CCLUT_TABLE_NAME)
     if (cclutMockTablePos > 0)
         if (cclut_mockTables->tables[cclutMockTablePos].isFinalized)
+            set cclutMockTableName = cclut_mockTables->tables[cclutMockTablePos].replaceName
             ;Drop the table if it was already added
+            ;For group1 users, they are unable to drop Discern definitions for tables (even ones they created).  A
+            ;prompt program will execute to perform the drop for them.  RDB drops have no such restrictions, so that can
+            ;be done directly
+            if (cclut::cclutGetCurrentGroup(null) = 0)
+                set cclutParserText = concat(" drop table ", cclutMockTableName, " go")
+                call parser(cclutParserText)
+            else
+                free record cclut_drop_table_request
+                record cclut_drop_table_request (
+                  1 program_name = vc
+                  1 query_command = vc
+                  1 output_device = vc
+                  1 Is_printer = c1
+                  1 Is_Odbc = c1
+                  1 IsBlob = c1
+                  1 params = vc
+                  1 qual [*]
+                    2 parameter = vc
+                    2 data_type = c1
+                  1 blob_in = gvc
+                )
+                set cclut_drop_table_request->program_name = "cclut_drop_table_definition"
+                set cclut_drop_table_request->params = concat("^MINE^,^", cclutMockTableName, "^")
+                set stat = tdbexecute(3070000, 3070001, 3050002, "REC", cclut_drop_table_request, "REC",
```

Comment:
Why not 3202004, 3202004, 3050002. **Everyone** will have access to 3202004. I have no idea about 3070001.